minerl/env/core.py

```diff
@@ -668,8 +668,19 @@ def _init_mission(self):
                 num_retries += 1
                 if num_retries > MAX_WAIT:
                     raise socket.timeout()
-                self.log_error("Did not get an OK from Malmo; trying again.")
-                time.sleep(1)
+                elif self.log_shows_bind_exception():
+                    # Skips error handler to directly to abort.
+                    self.had_to_clean = True
+                    logger.error("Malmo server failed to bind to port, possibly due "
```

Comment:
It seems like it'd be good (either here or in the `log_shows_bind_exception()` method) to surface and print the actual text of the BindException encountered, just in case it happened for reasons other than port collision

Reply:
Good idea